### PR TITLE
Google Analytics (Android + iOS)

### PR DESCRIPTION
**Pull request by MatusevichMaxim**
STEE-594 && STEE-595

### CODE REVIEW

File: Sources/Steepshot/Steepshot.Android/Services/GAService.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        public static GAService Instance()
+        {
+            if (thisRef == null)
+                thisRef = new GAService();
+            return thisRef;
+        }
+
+        public void InitializeGAService(Context appContext = null)
+        {
+            GAInstance = GoogleAnalytics.GetInstance(appContext.ApplicationContext);
```
Comment (KorzunAV) [inline Sources/Steepshot/Steepshot.Android/Services/GAService.cs]: appContext may be null =>  NullReferenceException

File: Sources/Steepshot/Steepshot.Android/Services/GAService.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            GATracker = GAInstance.NewTracker(TrackingId);
+            GATracker.EnableExceptionReporting(true);
+            GATracker.EnableAdvertisingIdCollection(true);
+            GATracker.EnableAutoActivityTracking(true);
+        }
+
+        public void TrackAppPage(string pageNameToTrack)
+        {
+            GATracker.SetScreenName(pageNameToTrack);
+            GATracker.Send(new HitBuilders.ScreenViewBuilder().Build());
```
Comment (KorzunAV) [inline Sources/Steepshot/Steepshot.Android/Services/GAService.cs]: To many operations in one line.

File: Sources/Steepshot/Steepshot.Core/Services/IGAService.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
+namespace Steepshot.Core.Services
+{
+    public interface IGAService
```
Comment (KorzunAV) [inline Sources/Steepshot/Steepshot.Core/Services/IGAService.cs]: Is interface really needed?

**Comment by MatusevichMaxim** [comment]
_Fixed_

**Pull request merged**